docs/content/tutorial/assets/testing-assets.mdx

```diff
@@ -3,15 +3,30 @@ title: Testing Assets | Dagster
 description: Dagster enables you to unit-test individual assets and graphs of assets
 ---
 
-# Testing Assets
+# Testing assets
 
 Creating testable and verifiable data pipelines is one of the focuses of Dagster. We believe ensuring data quality is critical for managing the complexity of data systems. Here, we'll cover how to write unit tests for individual assets, as well as for graphs of assets together.
 
-## Testing the cereal asset definitions
+Let's go back to the assets we defined in the [prior section](/tutorial/assets/asset-graph#a-more-complex-asset-graph) and ensure that they work as expected by writing some unit tests.
 
-Let's go back to the assets we defined in the [prior section](/tutorial/assets/asset-graph#a-more-complex-asset-graph), and ensure that they work as expected by writing some unit tests.
+---
+
+## Testing the nabisco_cereals asset definition
+
+We'll start by writing a test for the `nabisco_cereals` asset definition, which filters the larger list of cereals down to the those that were manufactured by Nabisco.
+
+First, create a new file and name it `test_complex_asset_graph.py`.
+
+At the top of the file, import the `cereals` and `nabisco_cereals` assets:
+
+```python
+from complex_asset_graph import (
+    cereals,
```

Comment:
I don't think the `cereals` asset needs to be imported to test the `nabisco_cereals` asset.

Reply:
Also, nitpick: I think this can all go on one line.